src/Pages/ProductList/component/Product.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+import '../component/Product.scss'
+
+class Product extends Component {
+  constructor() {
+    super()
+    this.state = {
+      wishBtn: false,
+      reviewRateContainer: true,
+      price: '',
+    }
+  }
+
+  handleWishBtn = () => {
+    this.setState({
+      wishBtn: !this.state.wishBtn,
+    })
+  }
+
+  componentDidMount = () => {
+    const priceComma = this.props.product.price
+      .toString()
+      .replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+    if (this.props.product.review === 0) {
```

Comment:
boolean 속성값을 이용해 아래와 같이 작성해주시면 더 간단해지겠네요! ㅎㅎ
```suggestion
    if (!this.props.product.review) {
```

Reply:
수정했습니다 !!